test/expected/plan_ordered_append-11.out

```diff
@@ -353,34 +343,36 @@ ORDER BY time ASC LIMIT 1;
   time_bucket('1d',time), device_id, value
 FROM ordered_append
 ORDER BY 1 LIMIT 1;
-                                                                 QUERY PLAN                                                                 
---------------------------------------------------------------------------------------------------------------------------------------------
+                                  QUERY PLAN                                   
```

Comment:
This change bothers me. It seem like the optimization isn't applied. Looking at `timescaledb_set_rel_pathlist` I see that it does expect the parent to be in the append list to apply the `ts_sort_transform_optimization` optimization.